bin/offline-cluster.sh

```diff
@@ -9,36 +9,60 @@ set -x
 
 ls $ANSIBLE_DIR/inventory/offline
 
+if [ -f "$ANSIBLE_DIR/inventory/offline/hosts.ini" ]
+then
+        INVENTORY_FILE="$ANSIBLE_DIR/inventory/offline/hosts.ini"
+else
+        if [ -f "$ANSIBLE_DIR/inventory/offline/inventory.yml" ]
+        then
+                INVENTORY_FILE="$ANSIBLE_DIR/inventory/offline/inventory.yml"
+        else
+                {
+                        echo "no inventory file in ansible/inventory/offline/. please supply an inventory.yml or hosts.ini."
```

Comment:
Could you please adjust the indentation rules in your editor? I'm not sure if we've got this specified for this project. However, that much indentation is a bit too much... :wink:

Reply:
thanks for finding that... turns out my weird IDE config on shell scripts
fixed 👍